### PR TITLE
make w3c happy

### PR DESCRIPTION
**Pull request by SimonLab**
see #167

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -156,7 +156,7 @@ <h2 class="call-to-action">Let's Talk!</h2>
 
         <fieldset class="pure-group">
           <label for="budget">Budget:</label>
-          <select id="budget" name="budget" required>
+          <select size="5" id="budget" name="budget" required>
```
Comment (iteles) [inline views/index.html]: Out of curiosity, why would you choose 5 instead of 3 seeing as there are only 3 visible options?

Comment (SimonLab) [inline views/index.html]: I think there are [5 selects](https://github.com/dwyl/dwyl.github.io/blob/e4a60129fa5c2a1d2d6146ccdbdd04dfadd5aeed/views/index.html#L160-L164) or did I add some by error?

**Comment by nelsonic** [comment]
Suspense is *killing* me! :grin:

**Comment by SimonLab** [comment]
@iteles @nelsonic I don't think the size attribute is the right solution at the end, see my [last comment](https://github.com/dwyl/dwyl.github.io/issues/167#issuecomment-220291420)

**Comment by SimonLab** [comment]
I think I found the right solution see the [this comment](https://github.com/dwyl/dwyl.github.io/issues/167#issuecomment-220294837)

**Comment by iteles** [comment]
👍 I think this will do for now. Kind of keen to make the budget a required field though as otherwise we'll just have to email them to ask!

Thanks again @SimonLab.

**Pull request merged**